lib/import/admin.js

```diff
@@ -142,17 +150,57 @@ function save_features(feature_collection) {
 }
 
 /**
+ * Get hash of admin to population
+ * mutates `feature_collection`.
```

Comment:
I just meant that `feature_collection` shouldn't be passed in, which you fixed.

Please update the `jsdoc` comment, though (the thing about "mutates feature_collection", which was never true).